tests/static/escapedHashtags/invalid-xpath.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>Untitled Form</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/939C596B-AD0C-4BB8-8717-F00707F71909" uiVersion="1" version="1" name="Untitled Form">
+					<text />
+					<hidden />
+				</data>
+			</instance>
+			<bind vellum:nodeset="#form/text" nodeset="/data/text" type="xsd:string" vellum:relevant="#invalid/xpath (`#form/hidden`" relevant="(`#form/hidden`" />
```

Comment:
What do you think of removing the hashtag escape delimiters from the (invalid) non-`vellum:` attribute? Additionally, would it be possible to convert all hashtags to valid xpath sub-expressions?

So ``relevant="(`#form/hidden`"`` becomes ``relevant="(/data/hidden"``

That way invalid xpath (or at least anything that is not valid according to our parser) is sent through as close to valid xpath as possible. I can imagine a scenario where mobile supports a syntactical form that our parser does not recognize, and it would be nice if those expressions could be entered in the form builder and passed on to mobile without non-standard escaping characters.